lib/yam/agama/agama_base.pm

```diff
@@ -20,4 +20,10 @@ sub post_fail_hook {
     save_and_upload_log('journalctl -u agama-auto', "/tmp/agama-auto-log.txt");
 }
 
+sub post_run_hook {
+    # In lib/main_common.pm setting LIVETEST & LIVECD will trigger logic which set password to NULL
```

Comment:
I asked @rakoenig to create a pre_run_hook to set `linux` as password.
The fact that the live settings set to undef the password is not really the reason, the reason is that agama has `linux` password because is based on Cockpit which has that password, and here the reason is that we need to change password to the one for the installed system when the installation succeeds. 

Therefore you could remove this comments, it might confuse us, because when alp will be in OBS we might need to implement something from scratch for livecd that will not have those settings at all.